flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/Buffer.java

```diff
@@ -238,6 +238,10 @@
 	/**
 	 * Used to identify the type of data contained in the {@link Buffer} so that we can get
 	 * the information without deserializing the serialized data.
+	 *
+	 * Notes: Currently, one byte is used to serialize {@link DataType}, so the maximum number
```

Comment:
nit: <p>Notes:

Reply:
....one byte of the ordinal in {@link DataType} is written into {@link BufferResponse}, so...